server/api/templates/docs/pd.swagger.json

```diff
@@ -0,0 +1,809 @@
+                                "$ref": "#/definitions/LogEvent"
+                            }
+                        }
+                    },
+                    "500": {
+                        "description": "unexpected error"
+                    }
+                }
+            }
+        },
+        "/pd/api/v1/balancers": {
+            "get": {
+                "description": "Get all PD balancers.",
+                "responses": {
+                    "200": {
+                        "description": "A balancers object.",
+                        "schema": {
+                            "$ref": "#/definitions/Balancers"
+                        }
+                    },
+                    "500": {
+                        "description": "unexpected error"
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Version": {
+            "type": "object",
+            "properties": {
+                "version": {
+                    "type": "string",
+                }
+            }
+        },
+        "Balance": {
+            "type": "object",
+            "properties": {
+                "min-capacity-used-ratio": {
+                    "type": "number",
+                    "format": "double",
+                    "description": "For capacity balance. If the used ratio of one store is less than this value, it will never be used as a from store."
+                },
+                "max-capacity-used-ratio": {
+                    "type": "number",
+                    "format": "double",
+                    "description": "For capacity balance. If the used ratio of one store is greater than this value, it will never be used as a to store."
+                },
+                "max-sending-snap-count": {
+                    "type": "integer",
+                    "format": "uint64",
+                    "description": "For capacity balance. If the sending snapshot count of one storage is greater than this value, it will never be used as a from store."
+                },
+                "max-receiving-snap-count": {
+                    "type": "integer",
+                    "format": "uint64",
+                    "description": "For capacity balance. If the receiving snapshot count of one storage is greater than this value, it will never be used as a to store."
+                },
+                "max-leader-count": {
+                    "type": "integer",
+                    "format": "uint64",
+                    "description": "For leader balance. If the leader region count of one store is less than this value, it will never be used as a from store."
+                },
+                "max-diff-score-fraction": {
+                    "type": "number",
+                    "format": "double",
+                    "description": "If the new store and old store's diff scores are not beyond this value, the balancer will do nothing."
```

Comment:
If the difference of the scores between the new store and old store does not exceed this value, the balancer will do nothing.